src/ui/components/CardList/styles.scss

```diff
@@ -11,7 +14,7 @@
     padding: 0;
 
     > li {
```

Comment:
Side note: Would be nice to swap things like this out for distinct classes when the opportunity presents itself.